src/Controllers/UserManagementController.php

```diff
@@ -21,4 +17,35 @@ public static function show(int $id)
         }
     }
 
+    public function create():void
+    {
+        echo '
+            <h1>Rejestracja nowego użytkownika</h1>
+            <form action="http://localhost/praktyki24-25/public/index.php?controller=usermanagement&do=store" method="POST">
```

Comment:
Zamiast `http://localhost/praktyki24-25/public/index.php?controller=usermanagement&do=store`

powinieneś umieścić `/public/index.php?controller=usermanagement&do=store`

Weź pod uwagę albo nawet umieść program w innym folderze. Wtedy aplikacja przestaje działać ponieważ nie ma takiego folderu i musisz robić zmiany w kodzie. Podajac ścieżkę taką jaką ci podałem wyżej program nie wysypuje się